deployment/postgres-init-db/sql/tables/merlin/external_events/external_event.sql

```diff
@@ -0,0 +1,83 @@
+create table merlin.external_event (
+    key text not null,
+    event_type_name text not null,
+    source_key text not null,
+    derivation_group_name text not null,
+    start_time timestamp with time zone not null,
+    duration interval not null,
+    properties jsonb,
```

Comment:
This field name should probably be `metadata` to be consistent with the rest of our DB. 

Additionally, why does `event_type` not define the permitted schema for metadata for events of that type? (This comment applies to `external_source_type` and `external_source` as well)

Reply:
In response to the second part: the schema was something we have intentionally descoped. We are not introducing schemas for allowed metadata in sources and events until probably the next PR; our reasoning was that before we introduce an extremely restrictive schema upfront, we would first like to explore and introduce external events, and then add schemae. Going forward, we will be working that into the `external_source_type` and `external_event_type` tables, and introducing a JSON Schema.

Reply:
Could you clarify then why you included the `metadata` column to begin with, if the column that describes its schema is "future work"?

Reply:
The first iteration we did here with `metadata` and `properties` was meant to be completely free form, though because it is completely arbitrary (i.e., no schema to lock it down), and we know we want to add a schema in the future, following a discussion with Joe and Jonathan, we have determined that the path we should take is to remove `metadata` and `properties` (from `external_event`) in this release and only include them in the next/future batch that adds the schema. This way, we also don't have to completely redefine `metadata` or `properties` in the future. 

This is put into place in [remove metadata and properties](https://github.com/NASA-AMMOS/aerie/commit/c07a140f8ebbb4227fecee88ac7fb80219f22213).